about.html

```diff
@@ -11,54 +11,80 @@
 
 </head>
 
+<header>
+  <section class="header">
+      <img src="img/lambda-black.png" alt="lambda-black"> 
+      <nav class="navbar">
```

Comment:
Unless you are changing the individual styles of the <a> elements, you can omit the class names on these.  They should all be identical.